view-admin/src/components/common/JudgementModal/JudgementModal.tsx

```diff
@@ -57,11 +57,12 @@ const JudgementModal = ({
       const currentValue = newNumbers[currentBox];
       const prevValue = currentBox > 0 ? newNumbers[currentBox - 1] : null;
 
```

Comment:
```suggestion
      const newValue = currentValue + value;
      const newValueNum = parseInt(newValue);
```
このissueとは関係ないけど変数は一つにまとめたいかも